src/pages/utkast/les-utkast-side.tsx

```diff
@@ -63,6 +64,18 @@ export function LesUtkastSide() {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [utkast, erBeslutter]);
 
+	function erUtkastOppdatert(gjeldende: Vedtak, nytt: Vedtak) {
+		return (
+			gjeldende.begrunnelse !== nytt.begrunnelse ||
+			gjeldende.hovedmal !== nytt.hovedmal ||
+			gjeldende.innsatsgruppe !== nytt.innsatsgruppe ||
+			!(
+				gjeldende.opplysninger.length === nytt.opplysninger.length &&
```

Comment:
Kanskje bedre med _. isEqual. Da slipper man ekstra logikk for å se om arrayene er like